packages/foam-vscode/src/services/variable-resolver.ts

```diff
@@ -26,7 +28,7 @@ const knownFoamVariables = new Set([
 
 export class Resolver implements VariableResolver {
   private promises = new Map<string, Promise<string | undefined>>();
-
+  private foamTitle = null;
```

Comment:
I don't believe we need this anymore, am I missing something?

Reply:
The function `resolveFoamTitle` is still assigning to that variable if I'm not mistaken.